services/frontend/src/utils/models/Profile.js

```diff
@@ -0,0 +1,38 @@
+import uuidv4 from "uuid/v4";
+
+export default class Profile {
+    constructor({
+        id = uuidv4(),
+        name = "",
+        primaryRole = "",
+        contactEmail = "",
+        phone = "",
+        slackHandle = "",
+        rocketChatHandle = "",
+        userId = uuidv4(),
+        createdAt = new Date(),
+        updatedAt = new Date(),
+    }) {
+        this.id = id;
+        this.name = name;
+        this.primaryRole = primaryRole;
+        this.contactEmail = contactEmail;
+        this.phone = phone;
+        this.slackHandle = slackHandle;
+        this.rocketChatHandle = rocketChatHandle;
+        this.userId = userId;
+        this.createdAt = createdAt;
+        this.updatedAt = updatedAt;
+    }
+
+    /* Normalizes the list of Profiles that the API returns into a map of {ID -> Profile},
+     * with the skills processed to just their IDs, for appropriate use in the Redux store. */
+    static normalizeApiResultsForRedux(Profiles = []) {
```

Comment:
It'd have been nice if you had also copy-pasted over the test suite for this function.

Reply:
I planned to take a deeper look at the tests and commit them in my next pr, when I'm handling actually showing data on the page.

Reply:
Good enough for me :+1: